### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by mmswflow-upb**
### Pull Request Overview

This pull request adds the documentation of the project.

### Testing Strategy

This pull request was tested by Sakka Mohamad-Mario

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by:  Sakka Mohamad-Mario <mmswflow@gmail.com>

### CODE REVIEW

File: docs/project/2024/mohamad_mario.sakka/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# DIY Laptop Cooling Pad
+
+
+**Author**: SAKKA Mohamad-Mario \
+**GitHub Project Link**: <https://github.com/UPB-FILS-MA/project-mmswflow-upb.git>
```
Comment (alexandruradovici) [inline docs/project/2024/mohamad_mario.sakka/index.md:5]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-mmswflow-upb.git
```

File: docs/project/2024/mohamad_mario.sakka/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+The format is
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price                                                                                                                                                                                           |
+| --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/mohamad_mario.sakka/index.md:73]:
```suggestion
|-|-|-|
```

File: docs/project/2024/mohamad_mario.sakka/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+| [Breaboard Power Supply](https://static.optimusdigital.ro/50620-large_default/sursa-de-alimentare-pentru-breadboard.jpg) | Powering Circuit | [1 x 4.69 RON](https://www.optimusdigital.ro/ro/electronica-de-putere-stabilizatoare-liniare/61-sursa-de-alimentare-pentru-breadboard.html) |
+| [AC-DC 9V Adaptor](https://static.optimusdigital.ro/5950-thickbox_default/alimentator-de-9-v-1-a.jpg) | Powering Circuit | [1 x 9.99 RON](https://www.optimusdigital.ro/ro/electronica-de-putere-alimentatoare-priza/264-alimentator-de-9-v-1-a.html) |
+| [Cooling Pad Frame]() | Laptop Stand That Holds Fans | [unkown]() |
+| **TOTAL** | - | [**186.64 RON**]() |
+## Software
+
+| Library                                                                     | Description               | Usage                                           |
+| --------------------------------------------------------------------------- | ------------------------- | ----------------------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/mohamad_mario.sakka/index.md:92]:
```suggestion
| - | - | - |
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mohamad_mario.sakka/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## Architecture
+
+* The application on the laptop will act as a client, which will connect to the WIFI network hosted on the RP Pico W, and it will send commands to it through TCP to ensure reliable communication between the two. This app will be written in *Python*
+  using *Custom Tkinter & Socket* modules and it will be converted into an executable. The RP PICO W will come with a preset network name & password. 
+
+* The physical UI (buttons & LCD) will be used to start the server, increase/decrease fan speed, power on/off the setup, with all necessary info being displayed on the LCD.
+
+![example](project_architecture.png)
```
Comment (alexandruradovici) [inline docs/project/2024/mohamad_mario.sakka/index.md:26]: I think that this file is missing from the repository.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mohamad_mario.sakka/index.md (hunk 4 above)
Comment (mmswflow-upb) [reply docs/project/2024/mohamad_mario.sakka/index.md:26]: I've uploaded the picture.

File: docs/project/2024/mohamad_mario.sakka/index.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+# DIY Laptop Cooling Pad
+
```
Comment (alexandruradovici) [inline docs/project/2024/mohamad_mario.sakka/index.md:2]: Please add a one line description of the project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — DISMISSED** [review]

File: docs/project/2024/mohamad_mario.sakka/index.md (hunk 5 above)
Comment (mmswflow-upb) [reply docs/project/2024/mohamad_mario.sakka/index.md:3]: Done.

**Review by irina-b-dev — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Please fix the failed test.

**Comment by mmswflow-upb** [comment]
> Please fix the failed test.

I think the issue is fixed, I added the missing links that the test couldn't find in the bill of materials table.

**Review by alexandruradovici — DISMISSED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**